README.md

```diff
@@ -19,4 +19,107 @@ We're always open to PRs from the community. To submit one:
 3. Make your changes.
 4. Submit a PR with an explanation of what/why/cavaets/etc.
 
-We'll review your content and work with you to make sure the fix gets pushed out quickly.
\ No newline at end of file
+We'll review your content and work with you to make sure the fix gets pushed out quickly.
+
+## About
+
+A collection of playbooks to help deploy and manage the Trend Micro Deep Security Agent.
+
+## Requirements
```

Comment:
The following sections should be moved above the Support and Contribution sections.